### DISCUSSION TITLE
how to deploy ansible Multi-Master replication of FreeIPA

### DISCUSSION THREAD

**Discussion by awsmaythem** [freeipa/ansible-freeipa#1048] [General]
hello 
how to deploy Multi-Master replication of FreeIPA using ansible ?

**Comment by flo-renaud** [comment]
Hi,

On Tue, Feb 21, 2023 at 7:50 AM awsmaythem ***@***.***> wrote:

> hello
> how to Multi-Master replication of FreeIPA using ansible ?
>
Please refer to *Installing an Identity Management replica using an Ansible
playbook *[1]. Also note that FreeIPA supports only multi-master replication
*.*

flo

[1]
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/installing_identity_management/installing-an-identity-management-replica-using-an-ansible-playbook_installing-identity-management

**Comment by awsmaythem** [comment]
i mean for example 

> [root@ipa-test]#ipa-replica-manage list
> ipa01.test.local: master
> ipa02.test.local: master
> [root@ipa-test]#ipa-replica-manage list ipa01.test.local
> ipa02.test.local: replica
> [root@ipa-test]#ipa-replica-manage list ipa02.test.local
> ipa01.test.local: replica

for node 1 and node 2 is master and replica both nodes for high availability
